sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/DataSourceStrategy.scala

```diff
@@ -673,6 +675,25 @@ object DataSourceStrategy
     (nonconvertiblePredicates ++ unhandledPredicates, pushedFilters, handledFilters)
   }
 
+  protected[sql] def translateAggregate(
+      aggregates: AggregateExpression,
+      pushableColumn: PushableColumnBase): Option[AggregateFunc] = {
+    aggregates.aggregateFunction match {
+      case min @ aggregate.Min(pushableColumn(name)) =>
+        Some(Min(FieldReference(Seq(name)), min.dataType))
+      case max @ aggregate.Max(pushableColumn(name)) =>
+        Some(Max(FieldReference(Seq(name)), max.dataType))
+      case count: aggregate.Count =>
+        val columnName = count.children.head match {
+          // SELECT COUNT(*) FROM table is translated to SELECT 1 FROM table
+          case Literal(_, _) => "1"
```

Comment:
We can use `org.apache.spark.sql.connector.expressions.Literal`?

Reply:
Do you mean 
```
    case Literal(_, _) =>
        org.apache.spark.sql.connector.expressions.Expressions.literal(1).toString
```